app/jobs/regular/export_user_archive.rb

```diff
@@ -118,6 +118,14 @@ class ExportUserArchive < ::Jobs::Base
 
     def execute(args)
       @current_user = User.find_by(id: args[:user_id])
```

Comment:
I think if it's expected we always use `@requesting_user` in the rest of the file, then `@current_user` should not be an instance var

Reply:
Ok all good :+1: